Jellyfin/Views/VideoViewModel.cs

```diff
@@ -16,19 +20,26 @@ public sealed class VideoViewModel : BindableBase
     private readonly JellyfinApiClient _jellyfinApiClient;
     private readonly JellyfinSdkSettings _sdkClientSettings;
     private readonly MediaPlayerElement _playerElement;
+    private readonly AppSettings _appSettings;
+    private Guid _playingVideoId;
+    private string _playingSessionId;
+    private Timer _progressTimer;
 
-    public VideoViewModel(JellyfinApiClient jellyfinApiClient, JellyfinSdkSettings sdkClientSettings, MediaPlayerElement playerElement)
+    public VideoViewModel(JellyfinApiClient jellyfinApiClient, JellyfinSdkSettings sdkClientSettings, MediaPlayerElement playerElement, AppSettings appSettings)
     {
         _jellyfinApiClient = jellyfinApiClient;
         _sdkClientSettings = sdkClientSettings;
         _playerElement = playerElement;
+        _appSettings = appSettings;
     }
 
     public async void HandleParameters(Video.Parameters parameters)
     {
-        Guid videoId = parameters.VideoId;
+        Guid videoId = _playingVideoId = parameters.VideoId;
+
+        PlaybackInfoResponse playbackInfo = await _jellyfinApiClient.Items[videoId].PlaybackInfo.GetAsync();
```

Comment:
I think this should be a POST instead of a GET?

See: https://api.jellyfin.org/#tag/MediaInfo/operation/GetPostedPlaybackInfo

I'm working on building up the DeviceProfile now, but if I'm reading this correctly, it seems like the client sends the DeviceProfile in that call and then the server sends back a `TranscodingUrl` (not sure what it does for direct play) which appears to be the `master.m3u8` playlist.